Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Stage 1 - Build Stage
+FROM gradle:7.4.2-jdk17-alpine as build
+
+
+COPY . .
+
+# Assemble Application
+RUN --mount=type=secret,id=all-secrets GITHUB_ACCESS_TOKEN=$(cat /run/secrets/all-secrets | grep GITHUB_ACCESS_TOKEN | awk -F "=" '{print $2}') && \
+./gradlew assemble -PGITHUB_ACCESS_TOKEN=$GITHUB_ACCESS_TOKEN
```

Comment:
Same as https://github.com/department-of-veterans-affairs/lighthouse-di-demo-java-starterkit/commit/d48fecf8a622c3a397f37ddaf9416237efc17f28#